swan-lake/development-tutorials/run-in-the-cloud/code-to-cloud-deployment.md

```diff
@@ -21,6 +21,8 @@ To complete this tutorial, you need:
 1. <a href="https://www.docker.com/" target="_blank">Docker</a> installed and configured in your machine
 2. <a href="https://kubernetes.io/docs/tasks/tools/" target="_blank">Kubectl</a> installed and configured in a <a href="https://minikube.sigs.k8s.io/docs/start/" target="_blank">Kubernetes cluster</a>
 
+>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips.
```

Comment:
```suggestion
>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is because the Ballerina Docker image is not supported on Apple Silicon chips yet.
```